build.psm1

```diff
@@ -191,51 +167,48 @@ function Start-ScriptAnalyzerBuild
 
         $destinationDir = "$projectRoot\out\PSScriptAnalyzer"
         # this is normalizing case as well as selecting the proper location
-        if ( $Framework -eq "core" ) {
-            $destinationDirBinaries = "$destinationDir\coreclr"
-        }
-        elseif ($PSVersion -eq '3') {
+        if ($PSVersion -eq '3') {
```

Comment:
`$PSVersion` above is compared to an `[int]` rather than a `[string]` -- I know they work out the same though :)

Reply:
This `if/elseif/else` might be a good candidate for a `switch` maybe:

```powershell
switch ($PSVersion)
{
    3
    {
    }

    4
    {
    }

    5
    {
    }

    default
    {
    }
}

Reply:
Thanks, point taken. I'll change it to a switch for 3,4,5 and 6 and throw otherwise